components/backup/src/endpoint.rs

```diff
@@ -962,6 +966,24 @@ impl<E: Engine, R: RegionInfoProvider + Clone + 'static> Endpoint<E, R> {
             }
             return;
         }
+        // CausalTsProvider may cache tso. Flush the cached tso to avoid use them in future data.
```

Comment:
Suggest: Flush causal timestamp to make sure that future writes will have larger timestamps, and help TiKV-BR acuqire a `backup-ts` with intact data smaller than it. (Note that intactness is not fully ensured now, until the `safe-ts` of RawKV is implemented. TiKV-BR need a workaround by rewinding `backup-ts` to a small "safe interval").